cypress_shared/pages/unAuthorised.ts

```diff
@@ -0,0 +1,7 @@
+import Page from './page'
+
+export default class UnAuthorisedPage extends Page {
```

Comment:
Go with `NotAuthorisedPage`/`notAuthorised.ts` here to keep consistent with your previous commit

Reply:
renamed file